libs/str.h

```diff
@@ -47,6 +47,11 @@
   #endif
 #endif
 
+#ifdef NULL
+#undef NULL
+#define NULL 0
+#endif
```

Comment:
hu? at the very least make this BSD only

Reply:
Okay i'll do that because on Clang 13.0.0 NULL is define like ((void*)0)